src/requirements.txt

```diff
@@ -1,4 +1,6 @@
+pyproj==1.9.5.1
 numpy==1.11.1
 rasterio==0.36.0
 requests==2.10.0
+scipy==0.18
```

Comment:
It's not, but I left a note about it in the commit message which added it: 6e5a3644340cfc8e82118031aecc2e13545f3a5a.  Basically, I keep trying scipy functions, but don't end up implementing with them.  It's nice to have the library included to experiment without needing to rebuild the container.